modules/flatcar-linux/main.tf

```diff
@@ -0,0 +1,3 @@
+data "external" "flatcar_version" {
+  program = ["sh", "-c", "curl https://${var.flatcar_channel}.release.flatcar-linux.net/amd64-usr/current/version.txt | sed -n 's/FLATCAR_VERSION=\\(.*\\)$/{\"flatcar_version\": \"\\1\"}/p'"]
```

Comment:
I don't think this works for AWS china, correct?

Reply:
No, that's true. For China we don't push this file format. I have probably to create it.

Reply:
Going to add also the use case for China by pushing to S3 `version.txt`